src/modules/spots/useSpots.js

```diff
@@ -111,6 +111,20 @@ const useSpots = (props) => {
     return rootSpot;
   };
 
+  const getSpotByImageId = (imageId) => {
```

Comment:
This function can be simplified to the following. We are only allowing an image to in one Spot so we can just use `find`.

    const getSpotByImageId = (imageId) => {
       return Object.values(getActiveSpotsObj()).find(spot => {
          return spot.properties.images && spot.properties.images.find(image => image.id === imageId);
       });
    };